proposal/model-registry.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Model Registry
+---
+
+**Introduction**
+
+Model Registry in an ML DevOps environment is a central repository for storing metadata about ML models.
+
+During the development and deployment life cycle of an ML model, various artifacts and metadata are created or associated with the ML model.
+For example, a *Data Scientist* may create training data, parameters, metrics during development. During deployment, an *MLOps Engineer* may work with artifacts such as versioned model data. Model serving will generate metadata about hyper parameters and metrics from inference. 
```

Comment:
an *MLOps Engineer* may work with a versioned Model form repository in serving to provide inferencing on the Model.

Reply:
Fixed.